src/products/README.md

```diff
@@ -40,3 +40,39 @@ Updates the current inventory value for the product identified by `{productID}`.
 Returns details on all categories.
 ### GET /categories/id/{categoryID}
 Returns details on the category identified by `{categoryID}`.
+
+## Testing
+To run integration tests for the Products service a Python virtual environment is required. You must have Python 3.3+ installed on your system to run the commands below. The commands are written to be ran from the project root.
```

Comment:
```suggestion
To run integration tests for the Products service a Python virtual environment is required. You must have Python 3.8+ installed on your system to run the commands below. The commands are written to be ran from the project root.
```

Let's use 3.8.  Most of the Lambda runtime use 3.8 (some use 3.7) so let's keep it consistent.